### DISCUSSION TITLE
Is anyone available to help me configure a basic starter app with nextjs (app router/server actions) and supabase?

### DISCUSSION THREAD

**Discussion by MincePie** [supabase/supabase#16653] [Jobs Board]
I am struggling to follow this tutorial: https://supabase.com/docs/guides/auth/auth-helpers/nextjs and the Egghead twitter clone tutorial. 

I would like to hire someone to help me get a basic form submission working so that I can crud actions in tables. I'd like to be able to do the things shown in these tutorials. 

If anyone has time in the next day or two for a session - I'd be grateful for the help in figuring out how to get a repo working. 

Please can you respond with rates and availability. 

I'm the first to admit that I struggle to make sense of coding documentation. I have been trying to learn for 10 years (at least 5 hours /week) and have never made it to deployment on a single attempt. I would like to ask for help with the specific things that seem to be problems using these templates (only) to get crud working - in a vercel deployment environment. The egghead tutorial goes all the way through. If I could get it working, I wouldn't be seeking help here. I blended these two tutorials because the egghead tutorial doesn't sync email /password auth (where the starter on the supabase website does). Either way, I couldn't get the egghead tutorial working in production (i could get further iwth that in the local environment). In production, I was getting errors with quotation marks that I could not disapply in the config. I'd like help to steer through these issues.

thank you

**Comment by mikoalala** [comment]
Hi @MincePie.  Sounds like something I could help with. Send me a message at same username on reddit.
